app/src/main/java/com/example/sharingang/viewmodels/ItemsViewModel.kt

```diff
@@ -69,9 +76,7 @@ class ItemsViewModel @Inject constructor(
     val userItemsAndRequests: LiveData<List<Item>>
         get() = _userItemsAndRequests
 
-    private val _userSoldItems = MutableLiveData<List<Item>>()
-    val userSoldItems: LiveData<List<Item>>
-        get() = _userSoldItems
+//
```

Comment:
I think you can remove this comment here.

Reply:
Thanks I removed it